src/modules/displaymanager/main.py

```diff
@@ -403,6 +401,11 @@ def run():
     # setup lightdm
     if "lightdm" in displaymanagers:
         if have_dm("lightdm", root_mount_point):
+            global lightdm_conf_path
```

Comment:
yes, it's also used in [set_autologin()](https://github.com/calamares/calamares/blob/master/src/modules/displaymanager/main.py#L264) and was trying to avoid repetitive code...